packages/react-admin/src/mui/list/SingleFieldList.js

```diff
@@ -16,35 +21,106 @@ const styles = {
  *         <ChipField source="title" />
  *     </SingleFieldList>
  * </ReferenceManyField>
+ * 
+ * By default, it includes a link to the <Edit> page of the related record
+ * (`/books/:id` in the previous example).
+ *
+ * Set the linkType prop to "show" to link to the <Show> page instead.
```

Comment:
You should add that to the `List.md` doc, too.

Reply:
Done